### PR TITLE
doc: initiate the documentation for Technical Writer

### PR DESCRIPTION
**Pull request by benjaminParisel**
I initiate the document.

Covers #52

### CODE REVIEW

**Comment by tbouffard** [comment]
@benjaminParisel I have updated the content with a563abb and b509417. This is my review 😸

**Comment by alachambre** [comment]
Why do we need two files CONTRIBUTING.adoc? 🤔

File: docs/content/CONTRIBUTING.adoc [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+To do it, you need to use this following syntax:
+
+[source,adoc]
+----
+[.card-section]
+== Highlight
+
+[.card.card-index]
+--
+yourInternalOrExternalLink[[.card-title]#Card Title# [.card-body.card-content-overflow]#pass:q[You can write a little description here to display it in the card body.]#]
```
Comment (tbouffard) [inline docs/content/CONTRIBUTING.adoc:88]:
```suggestion
yourInternalOrExternalLink[[.card-title]#Card Title# [.card-body.card-content-overflow]#pass:q[You can write a short description here to display it in the card body.]#]
```

File: docs/content/CONTRIBUTING.adoc [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+--
```
Comment (tbouffard) [inline docs/content/CONTRIBUTING.adoc:89]: ⚠️ if this is updated, the screenshot should be updated as well

**Comment by benjaminParisel** [comment]
> Why do we need two files CONTRIBUTING.adoc? 🤔

One for contributing in this repository and one to link it in all content repo (it avoids duplication)

File: CONTRIBUTING.adoc [hunk 3]
```diff
@@ -17,8 +17,8 @@ Here is the expected commit format, please follow https://www.conventionalcommit
 
 == Pull request
 
-Each improvment, fix must be submit with a pull request. When a pull request is submit, a action is running to check if your pull request follow commit message format.
+Each improvement, fix must be submit with a pull request. When a pull request is submit, an action is running to check if your pull request follow commit message format.
```
Comment (alachambre) [inline CONTRIBUTING.adoc:20]:
```suggestion
Each improvement, fix must be submit with a pull request. When a pull request is submit, an action is run to check if the pull request follows the commit message format.
```

File: docs/content/CONTRIBUTING.adoc [hunk 4]
```diff
@@ -0,0 +1,175 @@
+= Contributing to the Bonita documentation
```
Comment (alachambre) [inline docs/content/CONTRIBUTING.adoc:1]:
```suggestion
= Contribute to the Bonita documentation
```

File: docs/content/CONTRIBUTING.adoc [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+
+You can put you content after metadata block
+----
+
+NOTE: This metadata information is useful to improve our visibility for search engine (SEO)
+
+
+== Asciidoc
+
+https://docs.asciidoctor.org/asciidoc/latest/[Asciidoc] is the markdown language chosen to write our documentation content. You can check the official documentation site to learn more on Asciidoc syntax.
```
Comment (alachambre) [inline docs/content/CONTRIBUTING.adoc:32]:
```suggestion
https://docs.asciidoctor.org/asciidoc/latest/[Asciidoctor] is the markdown language chosen to write our documentation content. You can check the official documentation site to learn more on Asciidoc syntax.
```

File: docs/content/CONTRIBUTING.adoc [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+
+WARNING: WIP
+
+
+== Antora
+
+We are using https://docs.antora.org/[Antora] to generate the Bonita documentation.
+
+WARNING: WIP
```
Comment (alachambre) [inline docs/content/CONTRIBUTING.adoc:41]: link to the architecture guide

File: docs/content/CONTRIBUTING.adoc [hunk 7]
```diff
@@ -0,0 +1,175 @@
...
+
+
+=== Tips and tricks
+
+We will give some tips and tricks already use and available in our documentation site.
```
Comment (alachambre) [inline docs/content/CONTRIBUTING.adoc:46]: should be removed IMHO

File: docs/content/CONTRIBUTING.adoc [hunk 8]
```diff
@@ -0,0 +1,175 @@
...
+Some attributes are available and define in antora.yml at repository root and could be use in content page.
+
+[source,yml]
+----
+asciidoc:
+  attributes:
+    varVersion: 2021.1
+----
+[example]
+ Use *{varVersion}* syntax to reference this attributes. It will be replaced by 2021.1 for each occurrence.
```
Comment (alachambre) [inline docs/content/CONTRIBUTING.adoc:112]: I would put a real example here. the * could be a bit confusing for non tech users.

File: docs/content/CONTRIBUTING.adoc [hunk 9]
```diff
@@ -0,0 +1,175 @@
...
+This allows user to download the source as attachments by providing a link directly for the code (no duplication between
+the actual examples and the documentation)
+
+
+
+== How to integrate a new component
+
+WARNING: WIP
+
+Add a new source in the antora playbook and reference a new branch
```
Comment (alachambre) [inline docs/content/CONTRIBUTING.adoc:162]:
```suggestion
To add a new component, you first need to create a Github repository which will contain the asciidoc sources of this component (i.e the documentation content). This repository must be organized according to the https://docs.antora.org/antora/2.3/organize-content-files/[Antora recommendations]. You can check existing repositories for working examples.
Then, you need to add a new source in the https://github.com/bonitasoft/bonitasoft.github.io/blob/master/antora-playbook.yml[Antora playbook]. A source references the URL of the repository and all the branches to retrieve.
Finally, you have to update the https://github.com/algolia/docsearch-configs/blob/master/configs/bonitasoft.json[search configuration] to make this component searchable.
```

File: CONTRIBUTING.adoc (hunk 3 above)
Comment (VishalThapaliya) [reply CONTRIBUTING.adoc:20]: For each improvement, the fix must be submitted with a pull request. When a pull request is submitted, an action is run to check if the pull request follows the commit message format.

File: docs/content/CONTRIBUTING.adoc [hunk 10]
```diff
@@ -0,0 +1,164 @@
...
+= A short title
+:description: Put just after title (without whitespace) a little description to explain what we can retrieve in this page.
+
+You can put you content after metadata block
+----
+
+NOTE: This metadata information is useful to improve our visibility for search engine (SEO)
+
+
+== Asciidoc
```
Comment (tbouffard) [inline docs/content/CONTRIBUTING.adoc:30]: ❓ markdown --> markup

File: docs/content/CONTRIBUTING.adoc [hunk 11]
```diff
@@ -0,0 +1,164 @@
...
+- modules/ROOT/taxonomy.adoc
+
+== How to integrate a new version of an existing component
+
+When we need to add a new version of one component, we need to:
+
+1. Create a new branch (in the repository related to the component
+1. To add a new component, you first need to create a Github repository which will contain the asciidoc sources of this component (i.e the documentation content). This repository must be organized according to the https://docs.antora.org/antora/2.3/organize-content-files/[Antora recommendations]. You can check existing repositories for working examples.
+1. Then, you need to add a new source in the https://github.com/bonitasoft/bonitasoft.github.io/blob/master/antora-playbook.yml[Antora playbook]. A source references the URL of the repository and all the branches to retrieve.
+1. Finally, you have to update the https://github.com/algolia/docsearch-configs/blob/master/configs/bonitasoft.json[search configuration] to make this component searchable.
```
Comment (tbouffard) [inline docs/content/CONTRIBUTING.adoc:164]:
```suggestion
. Create a new branch (in the repository related to the component
. To add a new component, you first need to create a Github repository which will contain the asciidoc sources of this component (i.e the documentation content). This repository must be organized according to the https://docs.antora.org/antora/2.3/organize-content-files/[Antora recommendations]. You can check existing repositories for working examples.
. Then, you need to add a new source in the https://github.com/bonitasoft/bonitasoft.github.io/blob/master/antora-playbook.yml[Antora playbook]. A source references the URL of the repository and all the branches to retrieve.
. Finally, you have to update the https://github.com/algolia/docsearch-configs/blob/master/configs/bonitasoft.json[search configuration] to make this component searchable.
```

File: docs/content/CONTRIBUTING.adoc (hunk 5 above)
Comment (tbouffard) [reply docs/content/CONTRIBUTING.adoc:32]: already applied

File: docs/content/CONTRIBUTING.adoc (hunk 2 above)
Comment (tbouffard) [reply docs/content/CONTRIBUTING.adoc:78]: not relevant anymore, the image has been removed to ease the maintenance

**Review by tbouffard — APPROVED** [review]
LGTM, remember this is a first start, improvements are welcome.

**Pull request merged**